Makefile

```diff
@@ -159,7 +159,7 @@ devenv: $(wildcard $(subst dir, devenv, $(TWO_DIRS_DEEP)))
 EMILY_CDK_SOURCE_FILES := $(wildcard $(subst dir, emily/cdk/lib, $(FIVE_DIRS_DEEP)))
 EMILY_CDK_SOURCE_FILES := $(wildcard $(subst dir, emily/bin/lib, $(FIVE_DIRS_DEEP))) $(EMILY_CDK_SOURCE_FILES)
 
-$(EMILY_CDK_TEMPLATE): $(INSTALL_TARGET) $(EMILY_OPENAPI_SPEC) $(EMILY_CDK_SOURCE_FILES)
+$(EMILY_CDK_TEMPLATE): $(INSTALL_TARGET) $(EMILY_CDK_SOURCE_FILES)
```

Comment:
Why was `$(EMILY_OPENAPI_SPEC)` removed?

Reply:
This no longer needed for making the local CDK template because the openapi specification is only necessary for making the apigateway instance and the lambda instance. In this make command we only make the table parts of the template.